k8s/demo/common/em-ccd-orchestrator/em-ccd-orchestrator-demo.yaml

```diff
@@ -0,0 +1,54 @@
+---
+apiVersion: flux.weave.works/v1beta1
+kind: HelmRelease
+metadata:
+  name: em-ccd-orchestrator-demo
+  namespace: em-ccd-orchestrator
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:prod-*
+spec:
+  releaseName: em-ccd-orchestrator-demo
+  timeout: 900
+  forceUpgrade: true
+  rollback:
+    enable: true
+    force: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: em-ccd-orchestrator
+    version: 0.1.4
+  values:
+    ### EM-CCD-ORCHESTRATOR SERVICES ###
+    em-stitching:
+      java:
+        releaseNameOverride: em-ccd-orchestrator-demo-stitching
+        environment:
+          SPRING_DATASOURCE_URL: 'jdbc:postgresql://em-ccd-orchestrator-demo-postgres-db:5432/postgres?ssl=disable'
```

Comment:
Demo Databases won't be accessible here, they should be . You should use postgresql pod with persistence in Demo.